docs/ReleaseNotes.md

```diff
@@ -3,6 +3,60 @@ id: ReleaseNotes
 title: Release Notes
 ---
 
+## 2020.9.0 - 9/29/2020
+
+### New Features
+
+#### Post-Processing: Bloom
+
+Have you always wanted to add a satisfying glow to your experiences? Or make your scene look like something out of Tron or a video game? Well, you’re in luck! 
+
+New to Enklu Cloud this release is our first in-editor, post-processing shader: **Bloom**! You can access Bloom settings in your project by navigating to the Scene Settings tab, on the left sidebar of the editor.  
+
+For more information on Bloom, check out our official guide [here](WebEditorBasics/PostProcessing).
+
+<img src="https://cdn.enklu.com/web-assets/bloom.gif" width="100%" alt="Bloom"/>
```

Comment:
This image should be added with the `![alt-text][path]` syntax, like `![Bloom](/img/product/editor/Bloom.gif)`. The underlying doc framework will handle creating all of the tags/style when it generates the md files into html.

Reply:
The way you've added the images to this repo, you'll have to use that markdown syntax James is talking about.  Though the other images for the release notes Grant did not add in this same way, that's why they are using the img tags.  Grant is too cool for us.  Using the markdown syntax is fine for now, though eventually we may want to swap everything over to Grant's coolness 😎